akka-actor/src/main/scala/akka/util/Reflect.scala

```diff
@@ -154,4 +154,25 @@ private[akka] object Reflect {
       }
     }
   }
+
+  /**
+   * INTERNAL API
+   */
+  private[akka] def findClassLoader(): ClassLoader = {
+    def findCaller(get: Int ⇒ Class[_]): ClassLoader =
+      Iterator.from(2 /*is the magic number, promise*/ ).map(get) dropWhile { c ⇒
```

Comment:
test passes and, I can see that is used only in:
```Reflect.getCallerClass map findCaller```

not sure how the magic works here....